client/src/components/ProfileEdit/index.js

```diff
@@ -194,6 +157,8 @@ const ProfileEdit = function () {
       },
       body: JSON.stringify(cleanForm),
     })
+      .then((res) => res.json())
+      .then(({ profile }) => setProfile(profile))
```

Comment:
This is a much better way of doing what I did in my PR for the Dog Sitter switch. Once this is merged we can use this instead!